src/Pages/LogIn/CheckAgreeContainer.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+
+class CheckAgreeContainer extends Component {
+  render() {
+    return (
+      <div className="CheckAgreeContainer">
+        <div className="checkboxContainer">
+          <input type="checkbox" />
+        </div>
+        <div className="agreeContainer">
+          <div className="userAgree">{this.props.check}</div>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
깔끔하네요!
비구조화만 습관화되면 더 깔끔할 것 같습니다 👍 
```suggestion
  render() {
  const { check } = this.props;
    return (
      <div className="CheckAgreeContainer">
        <div className="checkboxContainer">
          <input type="checkbox" />
        </div>
        <div className="agreeContainer">
          <div className="userAgree">{check}</div>
        </div>
      </div>
    );
  }
}
```

Reply:
수정 완료했습니다.
비구조화 할당을 할것을 생각하면서 state 의 키값을 네이밍하면 보다 직관적이고 가독성이 좋은 키 이름을 지을수 있을것 같다는 생각이 드네요😋
더 익숙해질 필요가 있는것 같습니다 감사합니다